packages/blade/codemods/typography-lineheight/transformers/typography-line-height.ts

```diff
@@ -0,0 +1,29 @@
+import type { Transform } from 'jscodeshift';
```

Comment:
the import statements will work without transpilation?

Reply:
Yep jscodeshift transpiles and runs them internally.